cmd/crowdsec-cli/alerts.go

```diff
@@ -22,14 +21,16 @@ var ActiveDecision bool
 
 func AlertsToTable(alerts *models.GetAlertsResponse) error {
 	if csConfig.Cscli.Output == "raw" {
-		fmt.Printf("id,Scope/Value,reason,version,message,country,as,events_count,created_at\n")
+		fmt.Printf("id,Scope/Value,reason,version,country,as,events_count,created_at\n")
```

Comment:
Could you please just remove the version in this mode because it's already in the reason.